PersonApi/serverless.yml

```diff
@@ -115,6 +115,45 @@ resources:
                           - Ref: 'AWS::Region'
                           - Ref: 'AWS::AccountId'
                           - 'table/Persons'
+  ## Steps required for AppConfig cloudformation
+    AppConfig:
+      Type: AWS::AppConfig::Application
+      Properties:
+        Name: $housing-tl/${self:provider.stage}/person-api"
+    DevEnvironment:
+      Type: AWS::AppConfig::Environment
+      Properties:
+        ApplicationId: !Ref EnvironmentApplication
+        Name: "Development"
+    ConfigurationProfile:
+      Type: AWS::AppConfig::ConfigurationProfile
+      Properties:
+        ApplicationId: !Ref ConfigurationApplication
+        Name: "PersonConfigurationProfile"
+        LocationUri: ${ssm:/person-api/${self:provider.stage}/arn}
+        Validators:
+          - Type: Feature Toggles
+            Content:
+            {
+              "CreatePerson": true
+            }
+    DeploymentStrategy:

```

Comment:
Is this actually creating the deployment? If so - it's probably ok to use `AppConfig.AllAtOnce` if that's an option?

Reply:
We can try it, seeing as it's a quicker strategy to use